slasher/db/indexed_attestations.go

```diff
@@ -163,31 +158,18 @@ func (db *Store) DoubleVotes(validatorIdx uint64, dataRoot []byte, origAtt *ethp
 
 // HasIndexedAttestation accepts an epoch and validator id and returns true if the indexed attestation exists.
 func (db *Store) HasIndexedAttestation(targetEpoch uint64, validatorID uint64) (bool, error) {
```

Comment:
Can this function be renamed or remodeled? It doesn't check if they have a specific indexed attestation. Just checks if they voted for that epoch or not.